build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 /*
  * SPDX-License-Identifier: Apache-2.0
  *
- * Copyright 2019-2020 Pierre Leresteux.
+ * Copyright 2019-2022 .
```

Comment:
There should still be the name of the owner of the copyright.
When it is a contribution from an individual on its own personal time, it is his name. Otherwise it is the legal name of the company hiring the people. Here there should probably be "CREATIVE DATA", or "Saagie SAS", or something like that, I don't know the registered name.

Reply:
Done.